docker-wrappers/SPRAS/README.md

```diff
@@ -0,0 +1,54 @@
+This image comes bundled with all of the necessary software packages to run SPRAS, and can be used for execution in distributed environments (like HTCondor).
+
+To create the Docker image, make sure you are in this repository's root directory, and from your terminal run:
+```
+docker build -t reedcompbio/spras -f docker-wrappers/SPRAS/Dockerfile .
+```
+
+This will copy the entire SPRAS repository into the container and install SPRAS with `pip`. As such, any changes you've made to the current SPRAS repository will be reflected in version of SPRAS installed in the container. Since SPRAS
+is being installed with `pip`, it's also possible to specify that you want development modules installed as well. If you're using the container for development and you want the optional `pre-commit` and `pytest` packages as well as a
+spras package that receives changes without re-installation, change the
+`pip` installation line to:
+```
+pip install -e .[dev]
+```
+This will cause changes to spras source code to update the intsalled package.
+
+**Note:** This image will build for the same platform that is native to your system (ie amd64 or arm64). If you need to run this in a remote environment like HTCondor that is almost certainly `amd64` but you're building from Apple Silicon, it is recommended to either modify the Dockerfile to pin the platform:
+```
+FROM --platform=linux/amd64 almalinux:9
+```
+
+Or to temporarily override your system's default by exporting the environment variable:
+```
+export DOCKER_DEFAULT_PLATFORM=linux/amd64
+```
+(This environment variable can then be cleared by running `unset DOCKER_DEFAULT_PLATFORM` to return your system to its default)
+
+
+## Testing
+
+The folder `docker-wrappers/SPRAS` also contains several files that can be used to test this container on HTCondor. To test the `spras` container
+in this environment, first login to an HTCondor Access Point (AP). Then, from the AP clone this repo:
+```
+git clone git@github.com:Reed-CompBio/spras.git
+```
+
+When you're ready to run SPRAS as an HTCondor workflow, navigate to the `spras/docker-wrappers/SPRAS` directory and run `condor_submit spras.sub`. This will
+submit SPRAS to HTCondor as a single job with as many cores as indicated by the `request_cpus` line in `spras.sub`, using `example_config.yaml` as the
+SPRAS configuration file. Note that you can alter the configuration file to test various workflows, but you should leave `unpack_singularity = true`,
+or it is likely the job will be unsuccessful. By default, the `example_config.yaml` runs everything except for `cytoscape`, which appears to fail periodically
+in HTCondor.
+
+To monitor the state of the job, you can run `condor_q` for a snapshot of how the job is doing, or you can run `condor_watch_q` if you want realtime updates (you
+should never run something like `watch condor_q`, because this places extraneous strain on the AP). Upon completion, the `output` directory from the workflow
```

Comment:
```suggestion
To monitor the state of the job, you can run `condor_q` for a snapshot of how the job is doing, or  `condor_watch_q` if you want realtime updates. Upon completion, the `output` directory from the workflow
```

I suggest removing the bad command. SPRAS users are highly unlikey to think of that and more likely to accidentally copy/paste the wrong command if it is here than think of it on their own.